### ISSUE TITLE
[DDC] Improve upgrade docs

### ISSUE THREAD

**Issue by joaofnfernandes** [docker/docker.github.io#814]
We need to document process to upgrade UCP, DTR, and underlying engine.
- What's the recommended order to upgrade?
- Upgrade controllers then nodes? 
- Can this be automated ( done in parallel) or needs to be sequential ?
- Upgrades engines then ucp?
- Upgrade UCP then DTR ?

**Comment by vikstrous** [comment]
Some of this is a duplicate of https://github.com/docker/docker.github.io/issues/1744

**Comment by joaofnfernandes** [comment]
Let's treat it as a duplicate for now and close this.
